data/json/vitamin.json

```diff
@@ -6,7 +6,7 @@
     "name": { "str": "Calcium" },
     "deficiency": "hypocalcemia",
     "min": -12000,
-    "rate": "15 m",
+    "rate": "90 m",
```

Comment:
AFAIU these rates should've been tuned to represent IRL values to be synced with RDA vitamin values of all the comestibles. Were they previously? Are they now? Don't get me wrong, I like this PR and will enjoy a playthrough with realistic vitamin requirements. But I wouldn't like if they were irrelevant again.

Reply:
Maybe just change initial values?

Reply:
rough aproximations of reasonably realistic values, I'm not gettign a PDH to make a CDDA PR, by default it took 90 days to lose all nutrition one started with, I simply multiplied it, later I will look into what drugs should do to vitamin levels, make them drain them more rapidly while under their effects, this should keep them relevant

Reply:
Sorry, there are just too many comments to read here. And part of them must obsoleted by later ones. Please try keeping the original description relevant so that people don't have to read the entire history. Anyway, I'm not the one who will decide if this gets merged, just trying to help. Resolve this thread if you feel it should be.